test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -2020,4 +2022,143 @@ public void testSummarySelection() throws Exception {
     // check that there are two files, with none having extra summary info
     assertMatches(output, "(?sm).*^.*total[:]2[,]\\s+missing[:]0[,]\\s+extra[:]0.*$.*");
   }
+
```

Comment:
Very nice test!!

Reply:
Wish i could take credit, but this was already in the test :)

Reply:
Oh I meant tests, all of them together are a good verification of the new options.

Reply:
Thanks!